src/Unirest/Request.php

```diff
@@ -613,8 +624,8 @@ public static function send($method, $url, $body = null, $headers = array(), $us
      */
     private static function sleep($seconds) {
         $secs = (int) $seconds;
-        // the fraction part of the $seconds will always be less than 1 sec
-        $microSecs  = ($seconds - $secs) * 1000000;
+        // the fraction part of the $seconds will always be less than 1 sec, extracting micro seconds
+        $microSecs  = (int) (($seconds - $secs) * 1000000);
```

Comment:
Question: Is this a bug fix since this change is not part of the forceful retries feature? If yes then what is it fixing?

Reply:
This unirest dependency is now being used in sdks that require php 8.1, and in php 8.1 implicit type casting raise runtime failures.
$microSecs was previously float|int and was passed in `usleep()` which takes `int` argument.